src/Command/Output.php

```diff
@@ -18,4 +18,6 @@ public function isVerbose(): bool;
 
 	public function isDebug(): bool;
 
+	public function isDecorated(): bool;
```

Comment:
Please also add `isVeryVerbose()`, I realized it's needed a few days ago. Thanks :)

Reply:
Done!